bluemira/equilibria/coils/_tools.py

```diff
@@ -123,4 +125,4 @@ def get_max_current(dx: float, dz: float, j_max: float) -> float:
     -------
     Maximum current [A]
     """
-    return abs(j_max * (4 * dx * dz))
+    return np.abs(j_max * (4 * dx * dz))
```

Comment:
this is actually not needed:
![image](https://github.com/Fusion-Power-Plant-Framework/bluemira/assets/81617086/c50488ec-2b37-4a55-b6a6-8d1f512cf5ef)